### PR TITLE
Make speed settings in game propagate

### PR DESCRIPTION
**Pull request by sofroniewn**
Make settings in game for player and camera speed propagate through to movement functions automatically

### CODE REVIEW

File: freemove.js [hunk 1]
```diff
@@ -12,20 +12,20 @@ Freemove.prototype.compute = function(keys, angle) {
...
-    if (keymap.angle[0] in keys) this.velocity.angle += this.speed
-    if (keymap.angle[1] in keys) this.velocity.angle -= this.speed
+    if (keymap.angle[0] in keys) this.velocity.angle += this.speed.angle
+    if (keymap.angle[1] in keys) this.velocity.angle -= this.speed.angle
   }
 
   if (keymap.scale) {
-    if (keymap.scale[0] in keys) this.velocity.scale += this.speed*.01
-    if (keymap.scale[1] in keys) this.velocity.scale -= this.speed*.01
+    if (keymap.scale[0] in keys) this.velocity.scale += this.speed.position*.01
```
Comment (freeman-lab) [inline freemove.js]: If you're going to break up speed, might as well make a `speed.scale` no? and do the `*.01` that way?

**Comment by freeman-lab** [comment]
LGTM code-wise (other than adding `scale`), did you test with non-yoked mode as well?

File: freemove.js (hunk 1 above)
Comment (sofroniewn) [inline freemove.js]: yup considered it - will change now

**Comment by sofroniewn** [comment]
Yup - tested non-yoked, will add the scale speed and then merge

**Pull request merged**